people.md

```diff
@@ -6,7 +6,7 @@ collection: people
 
 | Name            | Contact Address                                                                                                     |
 | Jason           | [jhemann@northeastern.edu](mailto:jhemann@northeastern.edu)/[jason.hemann@gmail.com](mailto:jason.hemann@gmail.com) |
-| Terese Wang     | [wang.haoqi@northeastern.edu](mailto:wang.haoqi@northeastern.edu)                                                   |
-| Colton Donnelly | [donnelly.co@northeastern.edu](mailto:donnelly.co@northeastern.edu)                                                 |
+| Ryan Jung       | [wang.haoqi@northeastern.edu](mailto:jung.ry@northeastern.edu)                                                      |
```

Comment:
do we also wanna change the email from `wang.haoqi` to `jung.ry` and same with mine?

Reply:
TY! Nice catch ⚾